doc/TESTING.md

```diff
@@ -18,15 +18,35 @@ sudo apt update
 sudo apt install nicotine
 ```
 
-If you prefer to install a .deb package directly, you can [download one here](https://nightly.link/nicotine-plus/nicotine-plus/workflows/packaging/master/debian-package.zip).
+If you prefer to install a .deb package directly, you can [download one here](https://nightly.link/nicotine-plus/nicotine-plus/workflows/packaging/master/debian-package.zip) [[Info](https://nightly.link/nicotine-plus/nicotine-plus/workflows/packaging/master/debian-package)].
```

Comment:
Please enlighten us as to the intended benefit of providing these [Info] links?

My initial thought about the linked page is that it doesn't seem to provide anything relevant to the remit of a tester (other than perhaps, the 'Direct download of artifact' link).

Reply:
The reason for providing the info links is that it's possible to quickly navigate to, for example, https://github.com/nicotine-plus/nicotine-plus/actions/runs/2351490898#artifacts (the "View run" link). It helped me identify the unstable build's commit hash that I wanted to reference (something other testers may want to do). Maybe there are other ways to figure this out. I didn't look too deeply into the information those links provide, so maybe these Info links aren't as useful as I thought.